biaspaltsau_a/14.Ansible.Workshop/redmine.yaml

```diff
@@ -0,0 +1,33 @@
+  
+---
+- hosts: redmine
+  become: yes
+  pre_tasks:
+    - name: Redmine. Install packages
```

Comment:
This pre-task should be inside application role, please move

Reply:
corrected